blocks/editable/index.js

```diff
@@ -141,6 +127,9 @@ export default class Editable extends Component {
 
 	onInit() {
 		this.updateFocus();
+		if ( this.props.value ) {
```

Comment:
Will this cause the content to flicker in only after TinyMCE initializes? An advantage of the original implementation is that the content would be displayed even while TinyMCE initializes. Seems like we could recreate this by dangerously assigning the default value as inner HTML, which... should be safe?

Reply:
I don't see any flickering and I don't think it's a good idea. In fact, this allows us to drop the "shouldComponentUpdate/componentWillReceiveProps" entirely which simplifies the TinyMCE component a lot. see 2eb008dd

Reply:
The flicker is definitely there, and it's sensible to expect as such since we're not setting the content of the field until TinyMCE finishes initializing.

Master: https://cloudup.com/c3NdeDRaboZ
This branch: https://cloudup.com/cnJth3Fbhjg

Reply:
Gotcha, thanks for the screenshots. I found a way to fix while avoiding the sCU false 
7cd4940